### PR TITLE
Roll out BraveBlockScreenFingerprinting to 100% of Beta Desktop users

### LINKED ISSUE #390: Roll out BraveBlockScreenFingerprinting to 100% of Beta Desktop users
Now that we have successfully rolled out to 50% of Beta Desktop users, we will bring it to the remaining Beta Desktop users.

### PR DESCRIPTION
**Pull request by arthuredelstein**
Resolves #390

### CODE REVIEW

**Comment by arthuredelstein** [comment]
(See also #391 for PR to main branch)

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on Nightly as per https://github.com/brave/brave-variations/pull/382#issuecomment-1266401095.

**Pull request merged**
